research-software-directory/RSD-as-a-service
# 1194 mentions overview admins

## Add mentions overview for admins

Changes proposed in this pull request:

* Add a mentions overview for admins, where the mentions can also be edited
* Add anchor tags to the admin tabs

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Log in as admin, go to the admin section, go to the mentions overview
* Do some search queries
* Try to edit some of the mentions
* Check the styling of the admin tabs

Closes #1194

PR Checklist:
